lib/active_model/serializer/attributes.rb

```diff
@@ -14,10 +14,12 @@ module Attributes
 
         # Return the +attributes+ of +object+ as presented
         # by the serializer.
-        def attributes(requested_attrs = nil, reload = false)
+        def attributes(options = {}, reload = false)
+          requested_attrs = options[:only]
+          excepts = Array(options[:except])
```

Comment:
Right, here both are applied as filters. `only: [:a, :b], except: [:b]` is equivalent to `only: [:a]`. This seems reasonable to me, although I can imagine how someone could be confused by it. Should this raise if we receive both?